service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -649,7 +657,7 @@ EOF
 if [[ -n "${INSTANCE_CONTAINER}" ]]; then
 
 # Indicate the end of Terra customizations of the jupyter_notebook_config.py
-cat << EOF >> "${USER_BASH_PROFILE}"
+cat << EOF >> "${CONTAINER_NOTEBOOK_CONFIG}"
```

Comment:
This is a suspicious change to have in this PR. Can you confirm if this is intentional?